omod/src/main/web/dashboardwidgets/obsacrossencounters/obsacrossencounters.html

```diff
@@ -36,12 +36,14 @@
         </tr>
         <tr ng-if="$ctrl.headers.length == 0">
             <th class="column-header">Encounter</th>
+            <th ng-if="$ctrl.config.showEncounterType" class="column-header">Encounter Type</th>
```

Comment:
Encounter Type should be localized

Reply:
Ok, thanks @mogoodrich. I guess I should do the same for `Encounter`.

Reply:
Maybe to be more precisely, I would also add `Date` to `Encounter` if `Encounter Type` column is configured. Is this ok @mogoodrich, @mks-d ?

Reply:
Not sure I understand @Ruhanga ?

Reply:
@mogoodrich, I'm referring to i18n of `Encounter` on `line 38`. Also currently this column carries the encounter dates. How about explicitly naming this column to `Encounter Date` when the `Encounter Type` column is configured?

Reply:
I'd rename this to `showEncounterTypeName`.

@mogoodrich right now the encounter date is displayed under the 'Encounter' column, and we can keep it as such for backward compatibility even though I find the naming weird. I would like to suggest that we do something like that (at least when `showEncounterTypeName == true`):

| Encounter | Date             |
|-----------|------------------|
| Vitals    | 2020-01-03 16:05 |
| Triage    | 2020-01-03 20:15 |
| ...    | ... |

Or, maybe simpler to keep the backward compatibility:
| Encounter                    |
|------------------------------|
| (Vitals) 2020-01-03 16:05:00 |
| (Triage) 2020-01-03 20:15:00 |
| ... |

Reply:
Hmm... I thought I responded to this last week, but not seeing my response now.  Agree, just go ahead and change the "Encounter" column header to "Date", that makes more sense anyway.